pkgs/by-name/ep/epson-tm-t88vi/package.nix

```diff
@@ -0,0 +1,62 @@
+{
+  fetchzip,
+  lib,
+  stdenv,
+  cmake,
+  cups,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "epson-tm-t88vi";
+  version = "3.0.0.0";
+
+  src = fetchzip {
+    url = "https://download3.ebz.epson.net/dsc/f/03/00/15/35/42/b1a708bb8b21d7a68ae7394287db440974b68a0e/tmx-cups-src-ImpactReceipt-${finalAttrs.version}_pck_e.zip";
+    hash = "sha256-c6VpnNXYebkDkK9kcTZ/ILE8pD/qSWKCHYqkHV+WIkc=";
+  };
+
+  cmakeFlags = with lib.strings; [
```

Comment:
The agreed best method to fix this is to use substituteInPlace unless we need to patch baby cmake files. Also the extra long comment is not needed at all, that is something standard to patch, unfortunately.